src/screens/ProfileScreen/elements/LinkAndSocialMedia.tsx

```diff
@@ -51,7 +53,7 @@ const LinkAndSocialMedia: React.FC<LinkProps> = ({username}) => {
       <Text
         style={{
           color: COLORS.gray500,
-          fontWeight: '400',
+          fontFamily: fonts.inter[400],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fontWeight property has been replaced with fontFamily. This change is more flexible as it allows for using different font weights defined in the fonts object.
```diff
-          fontWeight: '400',
+          fontFamily: fonts.inter[400],
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->